### PR TITLE
Redesign: switch the color scheme

### LINKED ISSUE #6209: Redesign: Color scheme
## Feature Description

Update general color scheme based on [Figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=3211%3A70090).

- [ ] Update all colors

Note that this includes the widely used background colors of the entire workspace.

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by miina**
## Context
Does the following:
- changes the old theme to `theme.DEPRECATED_THEME` and starts using the new design system under `theme`.
- Switches some main colors to the new color scheme already, such as background colors and some font colors.

## Testing Instructions
1. Open the editor
2. See that things look "normal"

### QA
Even though this PR has some visual changes, there is no point doing QA for it since it mainly changes to the design-system and the specific sections of the workspace will be all addressed properly in separate issues.

- [x] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

### UAT

- [x] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testiing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #6209

### CODE REVIEW

File: assets/src/edit-story/app/media/media3p/attribution.js [hunk 1]
```diff
@@ -38,13 +38,14 @@ const AttributionPill = styled.div`
   display: flex;
   flex-wrap: nowrap;
   font-size: 12px;
-  color: ${({ theme }) => theme.colors.fg.white};
-  background-color: ${({ theme }) => rgba(theme.colors.bg.black, 0.7)};
+  color: ${({ theme }) => theme.DEPRECATED_THEME.colors.fg.white};
```
Comment (barklund) [inline assets/src/edit-story/app/media/media3p/attribution.js:41]: Wow, these definitely stand out! It just screams **_"UPDATE ME!!"._**

File: assets/src/edit-story/components/helpCenter/index.js [hunk 2]
```diff
@@ -74,7 +70,7 @@ export const HelpCenter = () => {
     <StyleSheetManager
       stylisPlugins={isRTL ? withRTLPlugins : withoutRTLPlugins}
     >
-      <ThemeProvider theme={dsTheme}>
+      <>
```
Comment (barklund) [inline assets/src/edit-story/components/helpCenter/index.js]: Is this fragment even needed here?

File: assets/src/edit-story/theme.js [hunk 3]
```diff
@@ -63,12 +68,14 @@ export const GlobalStyle = createGlobalStyle`
   }
 
   *::-webkit-scrollbar-track {
-    background: ${({ theme }) => rgba(theme.colors.bg.black, 0.1)};
+    background: ${({ theme }) =>
+      rgba(theme.DEPRECATED_THEME.colors.bg.black, 0.1)};
```
Comment (barklund) [inline assets/src/edit-story/theme.js:72]: Could we adjust the scrollbar colors here once and for all? They are as you pointed out way too faint. So change this to `theme.colors.bg.tertiary`, which should be a "Gray / 70". And also in the other places necessary to address scrollbars throughout.

**Review by barklund — APPROVED** [review]
This is actually a great list of files to work against to make sure, that we update everything in the editor in these two sprints (or actively decide not to update something). Bookmarked!

File: assets/src/edit-story/theme.js (hunk 3 above)
Comment (barklund) [reply assets/src/edit-story/theme.js:72]: So, no `rgba(... , .1)` here - just the tertiary gray as-is.

File: assets/src/edit-story/components/helpCenter/index.js (hunk 2 above)
Comment (miina) [reply assets/src/edit-story/components/helpCenter/index.js]: Eh, no :) Removed.

File: assets/src/edit-story/theme.js (hunk 3 above)
Comment (miina) [reply assets/src/edit-story/theme.js:72]: I think it's better to do this separately. There are multiple places in the code where scrollbar colors are defined currently and in some places, these seem to differ -- e.g. for the library vs zoom vs popup. So I think it's better not to hold off this PR for confirming and verifying the scrollbar colors -- we can add a separate issue that does that for all the scrollbars.

**Pull request merged**